src/agni/server/manager/UserManager.java

```diff
@@ -16,8 +16,69 @@ public UserManager(InfoSender infoSender,
     }
 
     @Override
-    public void userRequest(String ip, byte type) {
+    public void userRequest(String ip, byte type, String message) {
```

Comment:
Instead of `message`, `argument` would probably be a better name for that last parameter.